### DISCUSSION TITLE
Is the setting wrong?

### DISCUSSION THREAD

**Discussion by gatarib** [sanni/cartreader#575] [Need help]
Hello, I have assembled it halfway and turned it on,
but nothing is displayed on the monitor. Am I doing something wrong with the settings?
![IMG_20221021_202514291_HDR](https://user-images.githubusercontent.com/77560501/197188555-932db54a-0b97-4f4c-bcce-67a3e939d251.jpg)

**Comment by sanni** [comment]
Did you flash the Mega2560 through the Arduino IDE?
Also check that you have connected the display cables the right way.

**Comment by gatarib** [comment]
Thanks for getting back to me.

Did you flash the Mega2560 through the Arduino IDE?： 
・YES. I am writing in HW5

Also check that you have connected the display cables the right way.：
・Display cables are connected as indicated.

**Comment by cbarnes01** [comment]
Interesting I'm having a similar issue, but my display cables seemed to be keyed the other way.

**Reply by gatarib** [reply]
How do I deal with it?

I am reading this using translation software so there may be something wrong. Sorry.

**Comment by sanni** [comment]
Here is how it should connect:

![display_connect_1](https://user-images.githubusercontent.com/19169494/198100430-636e69bb-749a-4ab5-8bd6-de9af3daa278.jpg)

![display_connect_2](https://user-images.githubusercontent.com/19169494/198099547-41753b32-f668-4e24-bd23-d72d7d602323.jpg)

**Reply by gatarib** [reply]
Yes. I have connected it this way and the symptoms have not improved. Thanks.

**Reply by sanni** [reply]
Change Cart_Reader.ino like this:

````
//******************************************
// !!! CHOOSE HARDWARE VERSION !!!
//******************************************
// Remove // in front of the line with your hardware version
// #define HW5
// #define HW4
// #define HW3
// #define HW2
// #define HW1
#define SERIAL_MONITOR
````

Upload. Wait. Open Serial Monitor with settings: no line ending and Baud: 9600
See what it says.

![adgeagsfaffa](https://user-images.githubusercontent.com/19169494/198119484-f055f913-3d85-46c6-871d-0360ae0dfc54.jpg)

If it says SD error, check that SD is inserted. If it still says SD error check your soldering at these pins for mistakes:

![8r56uure](https://user-images.githubusercontent.com/19169494/198120191-55102d90-15fd-4db8-bb10-def99e10c776.jpg)

**Reply by gatarib** [reply]
Thanks for the advice. It does indeed say SD error. I tried re-soldering several times, but the result did not change.
Does this mean that the soldering is not working?
Should I try again with another board?
![cart reader-2](https://user-images.githubusercontent.com/77560501/198136571-3a55b3a7-94c2-4dfd-b8d6-5f7bc667ad8a.jpg)

**Reply by sanni** [reply]
With a multimeter set  to continuity test(the setting where it beeps when the two probes are touching) check that these 4 points(red to red, blue to blue, green to green, yellow to yellow) have contact.

![sfaagfffafffa](https://user-images.githubusercontent.com/19169494/198143271-cddc2230-fae2-495d-b682-fb4e76105cc6.jpg)

**Comment by gatarib** [comment]
All points tested OK for continuity!
![cart reader-3](https://user-images.githubusercontent.com/77560501/198150590-0f9ebccc-ec28-4d52-9ab9-b5dde4819617.jpg)
